package.json

```diff
@@ -22,5 +22,8 @@
   "bugs": {
     "url": "https://github.com/ringcentral/ringcentral-javascript/issues"
   },
-  "homepage": "https://github.com/ringcentral/ringcentral-javascript"
+  "homepage": "https://github.com/ringcentral/ringcentral-javascript",
+  "devDependencies": {
+    "eslint-plugin-ringcentral": "1.0.3"
```

Comment:
should it be there? https://github.com/ringcentral/ringcentral-javascript/blob/master/eslint-config-ringcentral/package.json

Reply:
I think yes, it's dependence, also we specify the working version